documentation/docs/libraries/java/how_to/_run_how_tos.md

```diff
@@ -1,25 +1,27 @@
-### Clone the Repository
 
-To run the examples, you will first need to clone the repository. You can do so by running the following command:
+### Set Up Your .env file
 
-```bash
-git clone git@github.com:iotaledger/wallet.rs.git
-```
-
-Then, move into the Java examples folder by running the following command:
+The code examples use a `.env` file to store variables. You can download
+the [example file](https://github.com/iotaledger/wallet.rs/blob/develop/wallet/.env.example) or create a new one with
+the following variables:
 
-```bash
-cd wallet.rs/bindings/java/examples
+```dotenv
+NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
+STRONGHOLD_PASSWORD="some_hopefully_secure_password"
+NODE_URL="http://localhost:14265"
+FAUCET_URL="http://localhost:8091/api/enqueue"
 ```
 
+You should update the `NODE_URL` and `FAUCET_URL` values to match the [Hornet node](#hornet-node) you want to use.
```

Comment:
For the shimmer testnet it would be `https://faucet.testnet.shimmer.network/api/enqueue` for the faucet, because if you just setup an own node the faucet is maybe not enabled or has probably no funds (if someone does that, then he probably also knows that he can use his url)